client/state/i18n/locale-suggestions/reducer.js

```diff
@@ -0,0 +1,50 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import {
+	I18N_LOCALE_SUGGESTIONS_REQUEST,
+	I18N_LOCALE_SUGGESTIONS_SUCCESS,
+	I18N_LOCALE_SUGGESTIONS_FAILURE,
+} from 'state/action-types';
+
+import { combineReducers } from 'state/utils';
+
+export const isFetching = ( state = false, action ) => {
```

Comment:
Note that when using the data-layer that we have middleware that handles dedupes, so checking for isFetching in the query component is no longer necessary. If we need state for a loader, we can usually avoid this by storing 3 states: null (not loaded), empty array (no locales), or a list of locales